objects/oPlayer/Keyboard_50.gml

```diff
@@ -0,0 +1,8 @@
+if(cooldown_2 == 0){
+	cooldown_2 = max_cooldown_2;
+	with(instance_create_layer(x,y,"Abilites",oArrow)){
```

Comment:
I think we should change the name of oArrow to oFireball before it's too late haha

Reply:
and the sprite too